### PR TITLE
Merge changes from v0.1.1 release into develop

### PR DESCRIPTION
**Pull request by patrickjahns**
This PR is to flow back the changes from 0.1.1 release into develop

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [0.1.1] - 2017-11-07
```
Comment (PVince81) [inline CHANGELOG.md]: 2017-11-13

Comment (patrickjahns) [reply CHANGELOG.md]: that's also something to consider - so the part with the final update for changelog should probably be done on the date we do the release?

Comment (PVince81) [reply CHANGELOG.md]: yeah

Comment (PVince81) [reply CHANGELOG.md]: can you adjust the date in this PR ?

**Comment by patrickjahns** [comment]
@PVince81 
adjusted

**Comment by PVince81** [comment]
conflicts?!

**Comment by patrickjahns** [comment]
The formatting of `info.xml` was changed in master (spaces to tabs: https://github.com/owncloud/impersonate/commit/ada4d26ed82df3cae7210009c1d4a269cbba1e2e#diff-224bb875c5d3cfcd7621135069013242 )

Adapt this branch or adapt master?

**Comment by PVince81** [comment]
@patrickjahns as you cannot (or are not supposed to) push directly to master, you can only adapt this branch.

**Comment by patrickjahns** [comment]
Any hints how to resolve this conflict without merging master into release and then merging it back. Or creating a rebase with totally new commits?

**Comment by patrickjahns** [comment]
Closing - rebased version #87

**Pull request closed without merging**